tests/baselines/reference/unicodeExtendedEscapesInStrings15_ES5.types

```diff
@@ -1,5 +1,5 @@
 === tests/cases/conformance/es6/unicodeExtendedEscapes/unicodeExtendedEscapesInStrings15_ES5.ts ===
 var x = "\u{abcd}\u{ef12}\u{3456}\u{7890}";
 >x : string
->"\u{abcd}\u{ef12}\u{3456}\u{7890}" : "ꯍ㑖碐"
+>"\u{abcd}\u{ef12}\u{3456}\u{7890}" : "\uABCD\uEF12\u3456\u7890"

```

Comment:
Consider adding support for `textSourceNode` from `StringLiteral` to `Identifier` so that you can just pull the raw text from the original source file. You would have to modify both `getTextOfNode` and `getLiteralTextOfNode` in emitter.ts to make a best effort to find the source file of the node if `currentSourceFile` is undefined, and otherwise just use the node's `text`.